projects/website/src/plots/PlotDriver.py

```diff
@@ -0,0 +1,323 @@
+                                continue 
+                            elif key_1.lower() == "Y2UpperBound".lower():
+                                job.Y2UpperBound = float(value_1)
+                                continue 
+                            elif key_1.lower() == "X1Label".lower():
+                                job.X1Label = value_1
+                                continue 
+                            elif key_1.lower() == "Y1Label".lower():
+                                job.Y1Label = value_1
+                                continue 
+                            elif key_1.lower() == "X2Label".lower():
+                                job.X2Label = value_1
+                                continue 
+                            elif key_1.lower() == "Y2Label".lower():
+                                job.Y2Label = value_1
+                                continue 
+                            elif key_1.lower() == "ExperimentalData".lower():
+                                job.experimentalData = value_1
+                                continue 
+                            elif key_1.lower() == "DataFileOverride".lower():
+                                job.dataFile = value_1
+                                continue 
+                            elif key_1.lower() == "DataPathOverride".lower():
+                                job.dataPath = value_1
+                                continue 
+                            elif key_1.lower() == "PFTFile".lower():
+                                job.PFTFile = value_1
+                                continue 
+                            elif key_1.lower() == "OutputOverride".lower():
+                                job.outputDir = value_1
+                                continue 
+                            elif key_1.lower() == "FontSize".lower():
+                                job.fontSize = int(value_1)
+                                continue 
+                            else:
+                                logging.warning("Unrecognized config directive: " + directive)   
+            fin.close()
+        except Exception as e:
+            logging.error("Ouch Something went wrong")
+
+    def execute(self):
+        for job in self.jobs:
+            if not job.ignore:
+                try:
+                    if self.jobname==job.name:
+                        job.logger=False
+                    else:
+                        job.logger=True
+                    if job.plotname=="ActionEvent":
+                        job.plotter.plot(job)
+                    self.jobname=job.name
+                except Exception as e:
+                    logging.error("Plotter couldn't plot job " + job.name + ". Check your config file line.")
+                    continue 
+        job.Reset()
+
+if __name__ == '__main__':
+    import sys
+    parser = argparse.ArgumentParser(description='Creation of Plots')
```

Comment:
I'm thinking you want to add a -j flag for threading control. These plots take for ever it would be nice if the program assumed a single thread by default but allowed you to run X threads with -jN